walk/config.go

```diff
@@ -58,7 +66,7 @@ type Configurer struct{}
 func (_ *Configurer) RegisterFlags(fs *flag.FlagSet, cmd string, c *config.Config) {
 	wc := &walkConfig{}
 	c.Exts[walkName] = wc
-	fs.Var(&gzflag.MultiFlag{Values: &wc.excludes}, "exclude", "Path to file or directory that should be ignored (may be repeated)")
+	fs.Var(&gzflag.MultiFlag{Values: &wc.excludes}, "exclude", "path.Match pattern that should be ignored (may be repeated)")
```

Comment:
Let's say `"pattern that should be ignored ...` without naming the specific implementation.

Reply:
done